### PR TITLE
feat(university-gateway): Updates to frontend and backend worker

### PR DESCRIPTION
**Pull request by berglindoma13**
# ...

Attach a link to issue if relevant

## What

Updates to worker that pulls data from all the universities and changes in design of frontend web page -> using /haskolanam-temp at the moment for a soft launch of the new look

## Why

New data schemas from universities called for updates to worker

## Screenshots / Gifs
university1
university2

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

**Review by busla — APPROVED** [review]
Reviewing `infra/src/uber-charts/` via devops, LGTM

**Review by jonarnarbriem — APPROVED** [review]

**Comment by RunarVestmann** [comment]
image

image

https://github.com/island-is/island.is/assets/43557895/11c01a59-4c62-416b-95ca-87d85797ceb2


After running this locally it looks like this for me on /s/haskolanam I'm guessing the screenshots above are from that same page and maybe I'm missing some data locally so it looks the same right?

File: apps/web/components/Organization/Wrapper/Themes/UniversityStudiesTheme/UniversityStudies.css.ts [hunk 1]
```diff
@@ -72,19 +162,19 @@ export const navigation = style({
...
 
-export const desktopTitleContainer = style({
-  ...themeUtils.responsiveStyle({
-    xl: {
-      display: 'flex',
-      justifyContent: 'center',
-    },
-  }),
-})
+// export const desktopTitleContainer = style({
```
Comment (RunarVestmann) [inline apps/web/components/Organization/Wrapper/Themes/UniversityStudiesTheme/UniversityStudies.css.ts:165]: Perhaps removing this commented code might be better than leaving it in here

File: apps/web/components/Organization/Wrapper/Themes/UniversityStudiesTheme/UniversityStudies.css.ts [hunk 2]
```diff
@@ -72,19 +162,19 @@ export const navigation = style({
...
+// })
 
-export const desktopTitle = style({
-  ...themeUtils.responsiveStyle({
-    xl: {
-      transform: 'translate(-150px, 165px)',
-    },
-  }),
-})
+// export const desktopTitle = style({
```
Comment (RunarVestmann) [inline apps/web/components/Organization/Wrapper/Themes/UniversityStudiesTheme/UniversityStudies.css.ts:174]: Perhaps removing this commented code might be better than leaving it in here

File: apps/web/components/Organization/Wrapper/Themes/UniversityStudiesTheme/UniversityStudiesFooter.tsx [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+          <Box display={'flex'} className={styles.footerLinkContainer}>
+            <LinkV2 href="/leit">
+              <Text color="blue600" fontWeight="semiBold">
+                <span style={{ textDecoration: 'underline' }}>Námsleiðir</span>
+              </Text>
+            </LinkV2>
+            <LinkV2 href="/leit">
+              <Text color="blue600" fontWeight="semiBold">
+                <span style={{ textDecoration: 'underline' }}>
+                  Stuðningsnet stúdenta
```
Comment (RunarVestmann) [inline apps/web/components/Organization/Wrapper/Themes/UniversityStudiesTheme/UniversityStudiesFooter.tsx:57]: Will this only be in Icelandic or should this be in english as well?

**Review by RunarVestmann — APPROVED** [review]
Few minor comments but I'll approve

**Pull request merged**